package.json

```diff
@@ -14,7 +14,7 @@
     "chai": "^4.0.1",
     "chai-as-promised": "^7.1.1",
     "dirty-chai": "^2.0.1",
-    "eslint": "^4.19.1",
+    "eslint": "^5.16.0",
```

Comment:
Should be part of a separate PR.

Reply:
It is: https://github.com/sendgrid/sendgrid-nodejs/pull/1063